### DISCUSSION TITLE
2023-06: OpenDAL 的文档即代码实践

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#403] [Announcements]
# 2023-06: OpenDAL 的文档即代码实践

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2023-06/

**Comment by lostz** [comment]
但是，我现在遇到一个问题，我对rust不熟悉，我想知道目前都支持哪些后端存储，怎么办，这方面是一点文档页没有

**Reply by Xuanwo** [reply]
在这里可以看到完整的列表 https://opendal.apache.org/docs/rust/opendal/services/index.html
